deployment/network-operator/README.md

```diff
@@ -218,15 +218,45 @@ helm upgrade -n network-operator  network-operator mellanox/network-operator --v
 
 >__NOTE__: `--devel` option required if you want to use the beta release
 
-### Return PODs to the drained nodes
 
->__NOTE__: this operation required only if **containerized OFED** is in use and if drain command
-was applied before to remove PODs from the cluster nodes
+### Restart PODs with containerized OFED drive 
+
+>__NOTE__: this operation required only if **containerized OFED** is in use
+
+Helm upgrade command will upgrade DaemonSet of the OFED driver to point to the new driver version,
+but will not remove or restart PODs with the driver on the nodes.
+The old OFED version will run on a node until you explicitly remove the driver POD or reboot the node.
+
+
+#### Remove PODs with secondary network from the node
+When containerized OFED driver reloaded on the node, all PODs which use secondary network based on
+NVIDIA Mellanox NICs will lose network interface in their containers.
+To prevent outage you need to remove all PODs which use secondary network from the node
+before you reload the driver POD.
+
+This can be done with node drain command:
+```
+kubectl drain <NODE_NAME> --pod-selector=<selector_for_pods>
+```
+
+#### Restart OFED driver POD
+Find OFED driver POD name for the node
+```
+kubectl get po -l driver-pod -o wide -A
+```
+
+Delete OFED driver POD from the node
+```
+kubectl delete po -n <DRIVER_NAMESPACE> <OFED_POD_NAME>
```

Comment:
Why not 
1. drain all nodes (not matter what pods we have)\
2. restart all ofed containers 
I don't get what it give us if you need to run it per node?

Reply:
It is required to implement rolling upgrade logic. In this case we don't need to remove all PODs with secondary networks from the cluster at the some time. I added some details to the doc itself.
We trying to achieve this flow:
1. Move PODs from node A to node B
2. Upgrade drive on node A
3. Return PODs to node A
4. Move PODs from node B to node A
5.  Upgrade drive on node B
So, we don't need to stop everething to upgrade the driver

Reply:
nit : po -> pod

Reply:
RE Moshe's comment, we could just provide both methods and let user decide.

Reply:
I replaced all naming shortcust in kubectl commands with full resources names.
Also, I rephrased some sentences to more explicitly express that both options are possible.